### PR TITLE
Added margin to EditText in AlertDialog

### PR DESCRIPTION
**Pull request by Swayam-Khare**
This PR closes issue #1323 
In this PR I tried to give proper margin to EditText field inside the MaterialAlertDialog. I have tested it on physical android device.
I have also added a dimen value inside dimen.xml file which contains the margin value in dp. Margin applied to EditText can be easily changed by changing the value of the dimen element.

### CODE REVIEW

**Comment by TheLastProject** [comment]
Makes sense, but this seems inconsistent with `showBalanceUpdateDialog` and `showInfoDialog` in `LoyaltyCardViewActivity.java`, which both use `R.dimen.alert_dialog_content_padding` (but seem to do it in a slightly different way? What a mess. And `createGroup` in `ManageGroupsActivity.java` uses a hardcoded 50).

I think we should just make all 4 dialogs use `R.dimen.alert_dialog_content_padding`, clean this all up while we're at it :)

**Comment by Swayam-Khare** [comment]
Ok, I will try to clean this up. So to be clear, all I have to do is to make sure that all four dialogs use the same R.dimen.alert_dialog_content_padding dimension?

**Comment by Swayam-Khare** [comment]
I made sure all four dialogs use `R.dimen.alert_dialog_content_padding`.

I noticed that `showInfoDialog` only has a TextView so in content padding is directly applied to the TextView. Whereas, in other three dialogs, we have EditTexts which require a parent container and margin.

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -26,4 +26,7 @@
     <dimen name="tileLetterFontSize">66sp</dimen>
     <dimen name="tileLetterFontSizeForShortcut">48dp</dimen>
     <dimen name="tileLetterImageSize">92dp</dimen>
+
+    <!--    Margin for EditText inside MaterialAlertDialog-->
+    <dimen name="dialog_margin">20dp</dimen>
```
Comment (obfusk) [inline app/src/main/res/values/dimens.xml:31]: I think this can be removed now that it's no longer used.

Comment (Swayam-Khare) [reply app/src/main/res/values/dimens.xml:31]: Yes, I forgot to remove it. I'll remove it

File: app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java [hunk 2]
```diff
@@ -491,10 +494,19 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
                     builder.setTitle(R.string.setBarcodeId);
                     final EditText input = new EditText(LoyaltyCardEditActivity.this);
                     input.setInputType(InputType.TYPE_CLASS_TEXT);
+
+                    FrameLayout container = new FrameLayout(LoyaltyCardEditActivity.this);
+                    FrameLayout.LayoutParams params = new  FrameLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT);
```
Comment (obfusk) [inline app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java:499]:
```suggestion
                    FrameLayout.LayoutParams params = new FrameLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT);
```

**Comment by obfusk** [comment]
I haven't had a chance to actually test the changes yet to see how they look, but the code LGTM :+1:
So I'd say it's ready for maintainer review now. Thanks!

**Comment by Swayam-Khare** [comment]
@obfusk Thanks 😄 

@TheLastProject Hello, can you review my recent commits on this PR please?

**Comment by obfusk** [comment]
@Swayam-Khare I'm sure she's already aware of this PR and will review it as soon as she has the time :)

**Comment by obfusk** [comment]
LGTM!

<details>
<summary>screenshot</summary>

Screenshot_20230602_160035

</details>

**Review by obfusk — APPROVED** [review]

**Comment by TheLastProject** [comment]
Looks good, thanks! And thank you @obfusk for helping review :)

**Pull request merged**